esx_service/utils/vmdk_utils.py

```diff
@@ -394,18 +417,22 @@ def get_datastore_name(datastore_url):
     # get_datastores() return a list of tuple
     # each tuple has format like (datastore_name, datastore_url, dockvol_path)
     res = [d[0] for d in get_datastores() if d[1] == datastore_url]
+    logging.debug("get_datastore_name: res=%s", res)
     return res[0] if res else None
 
+
 def get_datastore_url_from_config_path(config_path):
     """Returns datastore url in config_path """
     # path can be /vmfs/volumes/<datastore_url_name>/...
     # or /vmfs/volumes/datastore_name/...
     # so extract datastore_url_name:
```

Comment:
Not this change, the comment is confusing a little, if the format is the latter "/vmfs/volumes/datastore_name/  then this function is returning datastore_name and not the URL.